aksel.nav.no/website/components/website-modules/examples/__parts-inline/FormNavigationSimple.tsx

```diff
@@ -0,0 +1,38 @@
+import { ArrowLeftIcon, ArrowRightIcon } from "@navikt/aksel-icons";
+import { Button, HGrid, Hide, Show } from "@navikt/ds-react";
+
+const FormNavigationSimple = () => (
+  <HGrid
+    gap={{ xs: "space-16", sm: "space-32 space-16" }}
+    columns={{ xs: 1, sm: 2 }}
+    width={{ sm: "fit-content" }}
+  >
+    <Hide above="sm" asChild>
+      <Button
+        variant="primary"
+        icon={<ArrowRightIcon aria-hidden />}
+        iconPosition="right"
+      >
+        Neste steg
+      </Button>
+    </Hide>
+    <Button
+      variant="secondary"
```

Comment:
Back-button needs to have `type="button"` so that it doesn't trigger validation.

```suggestion
    <Button
      type="button"
      variant="secondary"
```